dashboard.php

```diff
@@ -26,23 +24,19 @@
         <a class="nav-text" href="index.php">Home Page</a>
             <form action="upload_intro.php" method="POST" enctype="multipart/form-data">
                 <h3>Intro:</h3>
-                    Title text:         <input name="title_text" type="text" value="<?php echo $title_array["title"] ?>"><br><br>
-                    Intro text:         <textarea name="intro_text" type="text" rows="10" cols="40"><?php echo $title_array["intro_text"] ?></textarea><br><br>
-                                        <input type="submit" name="Submit All" value="submit all">
+                <label>Title text:</label><br><input name="title_text" type="text" value="<?php echo $title_array["title"] ?>"><br><br>
```

Comment:
these lines are now too long, you should drop the inputs onto a different line from the labels